bears/general/SpaceConsistencyBear.py

```diff
@@ -38,7 +42,50 @@ def run(self,
         result_texts = []
         additional_info_texts = []
 
-        for line_number, line in enumerate(file, start=1):
+        def get_blanklines_nr_end():
```

Comment:
Can we come up with a better name for the function? This name is quite misleading.